src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php

```diff
@@ -325,18 +325,24 @@ public function testGenerateWithSpecialRouteName()
 
     public function testUrlEncoding()
     {
+        if (defined('PHP_QUERY_RFC3986')) {
```

Comment:
No you didn't do it for this file. See https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php#L328